lib/ukfile/pollqueue.c

```diff
@@ -133,18 +134,24 @@ uk_pollevent uk_pollq_assign_n(struct uk_pollq *q, uk_pollevent val, int one)
 {
 	uk_pollevent prev;
 	uk_pollevent set;
+#if CONFIG_LIBUKFILE_CHAINUPDATE
+	uk_pollevent clr;
```

Comment:
Feel free to disregard, but in my personal opinion, I find it cleaner to use `__maybe_unused` in cases like this.